src/DataAccess/DatabaseSettings.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Ticketing.DataAccess;
+
+public class DatabaseSettings
+{
+    public required string ConnectionString { get; set; }
+    public required int Timeout { get; set; }
+    public required int RetryCount { get; set; }
+    public required double RetryDelay { get; set; }
```

Comment:
For readability
```suggestion
    public required double RetryDelaySec { get; set; }
```

Also should it be `int` type instead of `double`?

Reply:
I am not quite sure this naming improves readability as I am using parameter names aligning with the [EnableRetryOnFailure](https://learn.microsoft.com/en-us/dotnet/api/microsoft.entityframeworkcore.infrastructure.sqlserverdbcontextoptionsbuilder.enableretryonfailure?view=efcore-7.0#microsoft-entityframeworkcore-infrastructure-sqlserverdbcontextoptionsbuilder-enableretryonfailure(system-int32-system-timespan-system-collections-generic-ienumerable((system-int32)))) method to setup a retry policy.

Reply:
Agree, let's keep `int` return type for both as the `int` accuracy should be enough.